docs/build/examples/meme-marketplace/step-3-run-hub-auth-server.md

```diff
@@ -17,7 +17,7 @@ Before starting the setup for the authentication server, you need to create API
 To do that, you’ll need:
 
 - [An account](https://docs.textile.io/hub/accounts/). This is your developer account on the Hub.
-- [A user group key](https://docs.textile.io/hub/app-apis/). This is how your users will be able to access your Hub APIs. Consider creating the key in an organization, not as your personal account, so that you can invite collaborators later. You only need to create one _user group key_ for all the users of your app.
+- [A user group key](https://docs.textile.io/hub/apis/). This is how your users will be able to access your Hub APIs. Consider creating the key in an organization, not as your personal account, so that you can invite collaborators later. You only need to create one _user group key_ for all the users of your app.
```

Comment:
`organization, not as your personal account` -> `organization account, not in your personal account`?

Reply:
I'm not touching copy in this PR. Literally just fixing links.